lib/drivers/docker/image.js

```diff
@@ -1,13 +1,11 @@
 'use strict';
 
-var cicada = require('strong-fork-cicada');
 var Container = require('./container');
-var debug = require('debug');
+var DeployReceiver = require('strong-deploy-receivers');
```

Comment:
not added as dep in package, probably that's why all the ci is failling

also, check that strong-deploy-receivers is in ci, and in the scrum-nodeops board